src/components/interactions/QuestionBoard.tsx

```diff
@@ -189,6 +206,24 @@ const QuestionBoard = ({ admin }: { admin?: boolean }) => {
       }
     : undefined;
 
+  const handleRemove = admin
+    ? (questionId: number) => {
+        if (socket) {
+          socket.emit(
+            "QuestionRemove",
+            { questionId },
+            (removedQuestion?: Question) => {
+              if (removedQuestion) {
```

Comment:
@KatieMacalintal Can we handle this in the`onQuestionRemoved` handler above instead of duplicating the filtering code here? To do so, I think we would need to send a message on the admin channel too (i.e., at line 136 in `socket.ts`)